modules/redis/redis.c

```diff
@@ -290,15 +294,82 @@ redis_worker_insert(LogThreadedDestDriver *s, LogMessage *msg)
       return LTR_ERROR;
     }
 
-  msg_debug("REDIS command sent",
-            evt_tag_str("driver", self->super.super.super.id),
-            evt_tag_str("command", full_command->str));
+  if (debug_flag)
+    {
+      GString *full_command = _argv_to_string(self);
+      msg_debug("REDIS command sent",
+                evt_tag_str("driver", self->super.super.super.id),
+                evt_tag_str("command", full_command->str));
+    }
 
   freeReplyObject(reply);
   scratch_buffers_reclaim_marked(marker);
   return LTR_SUCCESS;
 }
 
+static LogThreadedResult
+redis_worker_insert_batch_mode(LogThreadedDestDriver *s, LogMessage *msg)
+{
+  RedisDriver *self = (RedisDriver *)s;
+
+  ScratchBuffersMarker marker;
+  scratch_buffers_mark(&marker);
+
```

Comment:
This is unused.

Reply:
Added the missing reclaim functions. (`_fill_argv_from_template_list` is working with scratch buffers, same as the `_argv_to_string` debug function)